assets/src/models/detour.ts

```diff
@@ -1,6 +1,25 @@
 import { LatLngLiteral } from "leaflet"
 import { ShapePoint, Stop } from "../schedule"
 import { CreateDetourMachineInput } from "./createDetourMachine"
+import { array, Infer, nullable, number, string, type } from "superstruct"
+
+export const SimpleDetour = type({
+  route: string(),
+  direction: string(),
+  name: string(),
+  intersection: string(),
+  updated_at: number(),
+})
+
+export type SimpleDetour = Infer<typeof SimpleDetour>
+
+export const SimpleDetourData = type({
```

Comment:
Nit (non-blocking): How would you feel about renaming this to `SimpleDetourCollection` or `GroupedSimpleDetours` or something like that?

Reasoning: It's not clear from the name what the difference is between `SimpleDetour` and `SimpleDetourData`, or that `SimpleDetourData` is actually a collection of detours.